src/wallet/rpcwallet.cpp

```diff
@@ -5798,14 +5360,27 @@ UniValue sc_send_certificate(const UniValue& params, bool fHelp)
         // any check for upper threshold is left to cert processing
     }
 
+    //--------------------------------------------------------------------------
+    CBitcoinAddress fromaddress;
+    if (params.size() > 9)
+    {
+        inputString = params[9].get_str();
+        if (inputString != "*")
```

Comment:
To keep this command consistent with the syntax of other ones, I would prefer using the empty string `""` instead of `"*"`.